### PR TITLE
Integrate blockchain terminology updates

### LINKED ISSUE #1742: Update terminology of the `integrate-blockchain` category
The existing terminology regarding Lisk applications needs to be updated with the new terminology that we aligned on.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1742

### CODE REVIEW

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc [hunk 1]
```diff
@@ -20,31 +22,29 @@ Muhammad Talha <muhammad.talha@lightcurve.io>
...
-:url_sidechain_client: glossary.adoc#sidechain-client
-// footnotes
-:fn_sidechain_client_glossary: footnote:client[See xref:{url_sidechain_client}[Sidechain client] for more details.]
 
 // TODO: Update the page by uncommenting the hyperlinks once the updated pages are available.
 
 By default, Lisk exposes xref:{url_advanced_rpc_endpoints}[a range of endpoints] to communicate with a node.
 
-Apart from that, sidechain clients{fn_sidechain_client_glossary}, specifically xref:{url_intro_modules}[Modules] and xref:{url_intro_plugins}[Plugins] of a sidechain client can also expose endpoints as described in the guides about xref:{url_endpoints_modules}[implementing endpoints for a module] and xref:{url_endpoints_plugins}[creating an endpoint for a plugin], respectively.
+Apart from that, xref:{url_intro_modules}[Modules] and xref:{url_intro_plugins}[Plugins] of a bockchain client can also expose endpoints as described in the guides about xref:{url_endpoints_modules}[implementing endpoints for a module] and xref:{url_endpoints_plugins}[creating an endpoint for a plugin], respectively.
```
Comment (bobanm) [inline docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc]:
```suggestion
Apart from that, xref:{url_intro_modules}[Modules] and xref:{url_intro_plugins}[Plugins] of a blockchain client can also expose endpoints as described in the guides about xref:{url_endpoints_modules}[implementing endpoints for a module] and xref:{url_endpoints_plugins}[creating an endpoint for a plugin], respectively.
```

File: docs/modules/ROOT/pages/integrate-blockchain/integrate-ui/faucet-transfer.adoc [hunk 2]
```diff
@@ -226,7 +226,7 @@ To check if the account balance has increased, use the xref:{url_checkBalance}[a
 The *Transfer tokens* page shall provide an interface for users to transfer tokens to other accounts.
 To perform a token transfer, a user needs to enter the recipient address, the amount of tokens they want to send, and the transaction fee they want to pay.
 They also need to provide their passphrase to sign the transaction.
-Once the `Submit` button is clicked, the provided information is used to create and send the transaction to the Hello sApp.
+Once the `Submit` button is clicked, the provided information is used to create and sent the transaction to the Hello blockchain node.
```
Comment (bobanm) [inline docs/modules/ROOT/pages/integrate-blockchain/integrate-ui/faucet-transfer.adoc]: "sent" should remain in the present tense, as it is part of "[it]is used to create and send the transaction" sentence.

I personally prefer it like that. If you want to use the passive voice anyway, the other valid option could be:

```suggestion
Once the `Submit` button is clicked, the transaction is created from the provided information, and sent to the Hello blockchain node.
```

As a rule of thumb, it is always easier to read sentences in active voice. So perhaps the best option is:

```suggestion
Once a user clicks the `Submit` button, the provided information is used to create the transaction, and then send it to the Hello blockchain node.
```

**Review by bobanm — CHANGES_REQUESTED** [review]
This is all I could find... 😇

File: docs/modules/ROOT/pages/integrate-blockchain/integrate-ui/create-get-account.adoc [hunk 3]
```diff
@@ -17,7 +17,7 @@ Muhammad Talha <muhammad.talha@lightcurve.io>
 // Project URLs
 :url_start_client: build-blockchain/module/command.adoc#start-client
 
-With a UI template in place, on this page, you'll learn how to create a new account and how to fetch details of an account.
+On this page, you'll learn how to create a new account and how to fetch details of an account through a Frontend.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/integrate-blockchain/integrate-ui/create-get-account.adoc]:
```suggestion
On this page, you'll learn how to create a new account and how to fetch details of an account through a frontend.
```

File: docs/modules/ROOT/pages/integrate-blockchain/integrate-ui/faucet-transfer.adoc [hunk 4]
```diff
@@ -381,7 +381,7 @@ return (
                 <FixedMenuLayout />
                 <Container>
                     <h2>Send tokens</h2>
-                    <p>On this page you can send tokens to any address within the Hello sidechain.</p>
+                    <p>On this page you can send tokens to any address within the Hello World blockchain.</p>
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/integrate-blockchain/integrate-ui/faucet-transfer.adoc:384]: This reminds me, we have to make the change in the UI (hello_frontend & boilerplate) example in `lisk-sdk-examples` as well.

File: docs/modules/ROOT/pages/integrate-blockchain/subscribing-to-events.adoc [hunk 5]
```diff
@@ -64,19 +64,15 @@ This can be changed by updating the `*keepEventsForHeights*` property in the xre
...
 
-
-On this page, you'll learn about how the aforementioned events can be subscribed to or retrieved within the Lisk ecosystem.
-
 [[the-api-client]]
 == Subscribing to an RPC event
 
-//The xref:{url_references_elements_apiclient}[] simplifies sending API requests to a xref:{url_sidechain_glossary}[Sidechain application (sApp)].
-The API client simplifies the RPC event subscription process not only for a sidechain client{fn_sidechain_client_glossary} but also for external plugins and services. With the API client, RPC-based interfaces i.e. IPC and WS can be used to subscribe to *RPC events*. 
+The API client simplifies the RPC event subscription process not only for a blockchain client but also for external plugins and services. With the API client, RPC-based interfaces i.e. IPC and WS can be used to subscribe to *RPC events*.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/integrate-blockchain/subscribing-to-events.adoc]:
```suggestion
The API client simplifies the RPC event subscription process not only for a blockchain client but also for external plugins and services. 
With the API client, RPC-based interfaces i.e. IPC and WS can be used to subscribe to *RPC events*.
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looks good! However, the diagram for [Integrating a user interface](https://liskhq.github.io/lisk-docs/beta/integrate-blockchain/index.html#integrating-a-user-interface) needs an update as well. The page has also got depreciated terminologies.

**Review by bobanm — APPROVED** [review]

File: docs/modules/ROOT/pages/integrate-blockchain/integrate-ui/faucet-transfer.adoc (hunk 4 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/integrate-blockchain/integrate-ui/faucet-transfer.adoc:384]: I have created an issue: https://github.com/LiskHQ/lisk-sdk-examples/issues/316

**Review by TalhaMaliktz — APPROVED** [review]
Thanks, I think the diagram can use some alteration in terms of colors: Here's my take on the diagram. The rest of it LGTM 👍🏼 

Hello overview - Hello sApp (1)

**Pull request merged**